qucs-doc/report/EKV26/Makefile.am

```diff
@@ -44,9 +46,9 @@ SUFFIXES = .dvi .tex .pdf .ps
 	$(LATEX) -interaction=batchmode $<
 
 .tex.pdf:
```

Comment:
this rule (maybe others, too) should not be states so many times. that's what `include` is for.

as you need to touch all occurences of this rule in your commit, you could as well simply put `include common_tex.mk` (or something like that)... cf my comment in 641ce37e88f5c515288739e2983e71dfdb112590.